### PR TITLE
Use pymonkey to provide pip-custom-platform

### PR DESCRIPTION
**Pull request by asottile**
@chriskuehl

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [tox]
 project = pip_custom_platform
 # These should match the travis env list
-envlist = py26,py27,py33,py34,pypy,pypy3
+envlist = py27,py33,py34,pypy,pypy3
```
Comment (chriskuehl) [inline tox.ini]: py35? we'll probably want that soon.

File: pip_custom_platform/pymonkey.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+def pymonkey_argparse(argv):
+    def _add_platform_param(parser):
+        parser.add_argument(
+            '--platform',
+            help='Custom platform name (default is auto-detected)',
```
Comment (chriskuehl) [inline pip_custom_platform/pymonkey.py]: is there any way to see the default now easily? that was kind of nice

**Comment by chriskuehl** [comment]
\o/ looks pretty great to me!

File: tox.ini (hunk 1 above)
Comment (asottile) [inline tox.ini]: yeah, it requires some shenanigans in .travis.yml, but I can set that up soon

File: pip_custom_platform/pymonkey.py (hunk 2 above)
Comment (asottile) [inline pip_custom_platform/pymonkey.py]: yeah I can probably make a lazy object of some sort that displays.   The problem is I can't import distutils.util during argument parsing or the whole thing falls apart.  I'll make something in a followup branch

**Pull request merged**